src/build/mkrules/cflags.env.mk

```diff
@@ -41,7 +41,7 @@ CFLAGS += $(COMMONFLAGS) -mcpu=power7 -nostdinc -g -mno-vsx -mno-altivec\
 	  -ffunction-sections -fdata-sections -ffreestanding -mbig-endian
 ASMFLAGS += $(COMMONFLAGS) -mcpu=power7 -mbig-endian
 CXXFLAGS += $(CFLAGS) -nostdinc++ -fno-rtti -fno-exceptions -Wall \
-	    -fuse-cxa-atexit
+	    -fuse-cxa-atexit -std=gnu++11
```

Comment:
I'm pretty sure we're still using an older compiler for internal (non-openpower) builds so we'll likely need to do some more checks to make it compatible there.